frontend/src/components/forms/CreateLog.tsx

```diff
@@ -18,47 +18,104 @@ import {
   Modal,
   ModalBody,
   ModalContent,
-  ModalHeader,
   ModalOverlay,
-  ScaleFade,
+  ModalHeader,
   Text,
-  Textarea,
+  ScaleFade,
+  Textarea
 } from "@chakra-ui/react";
-
+import type { AlertStatus } from "@chakra-ui/react";
 import { AddIcon } from "@chakra-ui/icons";
 import { SingleDatepicker } from "chakra-dayzed-datepicker";
 import { Col, Row } from "react-bootstrap";
+import { AuthenticatedUser } from "../../types/AuthTypes";
+import UserAPIClient from "../../APIClients/UserAPIClient";
+import ResidentAPIClient from "../../APIClients/ResidentAPIClient";
+import { getLocalStorageObj } from "../../utils/LocalStorageUtils";
+import AUTHENTICATED_USER_KEY from "../../constants/AuthConstants";
+import LogRecordAPIClient from "../../APIClients/LogRecordAPIClient";
 import selectStyle from "../../theme/forms/selectStyles";
 import { singleDatePickerStyle } from "../../theme/forms/datePickerStyles";
+
+type Props = {
+  getRecords: (page_number: number) => Promise<void>;
+  countRecords: () => Promise<void>;
+  setUserPageNum: React.Dispatch<React.SetStateAction<number>>;
+};
+
+type NewSelectOptionType = {
+  label: string;
+  value: number;
+}
+
+type AlertData = {
+  status: AlertStatus;
+  description: string;
+}
+
+type AlertDataOptions = {
+  [key: string]: AlertData;
+}
+
 // Ideally we should be storing this information in the database
 const BUILDINGS = [
   { label: "144", value: "144 Erb St. West" },
   { label: "362", value: "362 Erb St. West" },
   { label: "402", value: "402 Erb St. West" },
 ];
 
+const ALERT_DATA: AlertDataOptions = {
```

Comment:
Helen merged in a PR that added a custom alert component. I'm okay with switching the alerts here after this pr is merged- this comment is more so a reminder to do that 🚗